test/test.js

```diff
@@ -9,11 +9,15 @@ if (!TOKEN) {
 	throw new Error('Please set TELEGRAM_TOKEN before running the tests!');
 }
 
+var params = {
+  proxy: process.env.Proxy
```

Comment:
I would either document this or leave it out in the tests. Or even better: could we use a public proxy to run a single test using the proxy?

Reply:
Public proxy? Could you recomend one?

Reply:
Maybe one of these would work: https://www.proxynova.com/proxy-server-list/ ?